release/air_tests/oom/tune_air_oom.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+# Test Ray Data bulk ingestion performance as the size of input files change.
+
+# Exit if any of the test commands fail.
+set -x -e pipeline
+
+NUM_IMAGES_PER_FILE="2048"
+NUM_FILES="16"
```

Comment:
Should this file be simplified given that you're only testing one NUM_FILES setting?